parallaxinc/Parallax-IDE
# handle overwrite cancel and remove overlay plugin

#### What's this PR do?

keeps the file save button options the same after an overwrite dialog is canceled, as well as some refactoring of the overlay statement into the router
#### Where should the reviewer start?

the save overlay view and the handlers
#### How should this be manually tested?

Create a new file then navigate away from it.  When the save dialog appears, try saving the file with an name already being used.  The click cancel on the overwrite dialog.  You should still have three option buttons on the save dialog.  Also, test the various other dialogs in the app.
#### What are the relevant tickets?

fixes #276
